DuckDuckGo/LargeOmniBarState.swift

```diff
@@ -27,7 +27,7 @@ struct LargeOmniBarState {
         let showBackButton: Bool = true
         let showForwardButton: Bool = true
         let showBookmarksButton: Bool = true
-        let showShareButton: Bool = false
+        let shoeAccessoryButton: Bool = false
```

Comment:
typo `shoe` -> `show`

Reply:
It's a shoe 🥿  accessory 😬 
Nice catch, thanks!